src/components/BookItem.jsx

```diff
@@ -13,41 +13,51 @@ const BookItem = () => {
 
   return (
     <div>
-      {isLoading && <div>Loading ........</div>}
-      {error && <div>Ooops! Something happend whiles fetching data</div>}
+      {isLoading && <div className="alert alert-info">Loading ........</div>}
+      {error && (
+        <div className="alert alert-danger">
+          Ooops! Something happend whiles fetching data
+        </div>
+      )}
       {!isLoading &&
         Object.entries(books).map(([itemId, book]) => (
-          <>
+          <div key={itemId} className="card-card">
             <div>
               <div>
-                <p>{book[0].type}</p>
-                <h3>{book[0].title}</h3>
-                <p>{book[0].author}</p>
-              </div>
-              <div>
-                <button type="button">Comment</button>
-                <button
-                  type="button"
-                  onClick={() => dispatch(deleteBook(itemId))}
-                >
-                  Remove
-                </button>
-                <button type="button">Edit</button>
+                <p className="trans">{book[0].category}</p>
+                <h4 className="trans">{book[0].title}</h4>
+                <p className="trans">{book[0].author}</p>
+                <div className="trans">
+                  <button type="button" className="btn">
+                    Comment
+                  </button>
+                  <button
+                    className="btn"
+                    type="button"
+                    onClick={() => dispatch(deleteBook(itemId))}
+                  >
+                    Remove
+                  </button>
+                  <button type="button" className="btn">
+                    Edit
+                  </button>
+                </div>
```

Comment:
You have done a good job with the styling of the app, however, there are few things to take into account in order for the app to look like the provided template:

- I suggested reviewing the sections marked blue in the screenshot below as the correct color scheme aen't correctly applied there
- I recommend reviewing the part marked fed as they are lacking the line separator and the comment, remove and edit texts aren't filled buttons but links

Zeplin template
<img width="498" alt="Screenshot 2023-06-16 180248" src="https://github.com/Theodoraldo/react-bookstore/assets/84930355/13751967-81f2-46c6-9f9d-530c2dc0c7ce">



Your design
<img width="580" alt="Screenshot 2023-06-16 180337" src="https://github.com/Theodoraldo/react-bookstore/assets/84930355/fb03f174-43a4-4b85-9af2-5a75b3b2f6d9">